packages/explorer/README.md

```diff
@@ -0,0 +1,30 @@
+### Explorer
+
+Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+
+## Getting Started
```

Comment:
once we have an exported binary, we should separate the tasks of using explorer (add package to project, run binary with env var) from developing explorer (installing package, running mprocs dev server)

Reply:
Updated changeset to include sections on installing explorer, example setup, and contributing/development guide. Lmk what you think